lock-api/src/main/java/com/palantir/lock/client/LockLeaseService.java

```diff
@@ -88,6 +90,26 @@ StartTransactionResponseV4 startTransactions(int batchSize) {
                 lease);
     }
 
+    StartTransactionResponseV5 startTransactionsWithWatches(OptionalLong version, int batchSize) {
+        ConjureStartTransactionsRequest request = ConjureStartTransactionsRequest.builder()
+                .requestorId(clientId)
+                .requestId(UUID.randomUUID())
+                .numTransactions(batchSize)
+                .lastKnownVersion(version.isPresent() ? Optional.of(version.getAsLong()) : Optional.empty())
```

Comment:
this is weird? but I can't think of another way :S

Reply:
why even have it be `OptionalLong` if you're going to have to box unbox everywhere?

Reply:
Fair, had to make it Optional<Long> to avoid breaking wire format and failed to update this